src/tracks/playabletrack/wavetrack/ui/WaveformView.cpp

```diff
@@ -988,7 +988,7 @@ void WaveformView::DoDraw(TrackPanelDrawingContext &context, size_t channel,
    channel = (track.IsLeader() ? 0 : 1);
 
    // TODO wide wave tracks -- remove this workaround
-   auto pLeader = *track.GetOwner()->Find(&track);
```

Comment:
Is this the line that was crashing? To my wobbly understanding returns `GetOwner()` and `GetHolder()` a pointer to the same object, only the former a the lock of a weak pointer and the latter, a raw pointer. But that wouldn't explain why this change would fix the crash.

Reply:
There is some complication during recording.  RegisterPendungChangedTrack sets up one or two tracks for which owner and holder differ.